src/main/java/com/thevoxelbox/voxelsniper/util/io/FileUtils.java

```diff
@@ -0,0 +1,54 @@
+package com.thevoxelbox.voxelsniper.util.io;
+
+import com.sk89q.worldedit.internal.util.LogManagerCompat;
+import com.thevoxelbox.voxelsniper.VoxelSniperPlugin;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class FileUtils {
```

Comment:
Is there a particular reason why this class was added, instead of extracting the file with MainUtil#copyFile from Fawe?

Reply:
This is due to FAWE specific usage here:

https://github.com/IntellectualSites/FastAsyncWorldEdit/blob/main/worldedit-core/src/main/java/com/fastasyncworldedit/core/util/MainUtil.java#L562